14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md

```diff
@@ -25,6 +25,22 @@ This page is a work in progress. It will be updated as the software evolves.
 }
 ```
 
+**Manifest with empty element**
+Create a element with a default export and load the element like this:
+
+```typescript
+const section : ManifestSection = {
+    "type": "section",
+    "alias": "Empty.Section",
+    "name" : 'Empty Section',
+    element : () => import('./empty-section.element.js'),
+    meta : {
+        label : 'Foo',
+        pathname : 'empty-section'
+    }
+}
```

Comment:
I keep trying to add this example in umbraco-package.json however I cannot find a way to make this code work 🤔 

Can you please provide a full example from your package manifest to see how one can reproduce this on a new installation?

Reply:
Ohh, sorry, this was added using a ”entity point” so it’s actually typescript code. Sorry for the lack of context.

I’ll see if I can update it to make more sense.